src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -137,6 +158,15 @@ class RemoteFileHandler:
 
     def __init__(self, transfer_service: Optional[Any] = None):
         self._transfer_service = transfer_service
+        self._service_type = (
+            ServiceType.PIM
+            if isinstance(transfer_service, PimFileTransferService)
+            else None
+        )
+
+    def is_configured(self):
+        if self._service_type == ServiceType.PIM:
+            return pypim.is_configured()
```

Comment:
`is_configured()` check is generic now. It is based on supported file transfer service types.

Reply:
Removed.